brands/amenity/bank.json

```diff
@@ -3353,6 +3353,16 @@
       "name": "HSBC"
     }
   },
+  "amenity/bank|HSBC UK": {
+    "countryCodes": ["gb"],
+    "tags": {
+      "amenity": "bank",
+      "brand": "HSBC",
+      "brand:wikidata": "Q5413890",
+      "brand:wikipedia": "en:HSBC",
```

Comment:
[Q5413890](https://www.wikidata.org/wiki/Q5413890) and “[HSBC Bank (Europe)](https://en.wikipedia.org/wiki/HSBC_Bank_%28Europe%29)” are for the UK-based European subsidiary, which is still named “HSBC”.

[Q64767453](https://www.wikidata.org/wiki/Q64767453) has been created for the UK-based UK retail banking subsidiary, which apparently has long been officially named HSBC UK but which now brands itself as such. The English Wikipedia only has a section on this sub-subsidiary, not a full article. So if `brand:wikidata` is changed to `Q64767453`, then `brand:wikipedia` should be removed for consistency.

There are Wikidata items for other national subsidiaries, but there’s no need to create separate entries for those subsidiaries, because their retail branches don’t carry national branding like the UK branches do.